src/index.html

```diff
@@ -22,6 +25,374 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple.html"
```

Comment:
The `href` attribute should lead to an actual page or be a placeholder like `#`. The `.html` extension should not be included in fragment identifiers.